stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/ui/common/helper/DownloadAttachmentUriGenerator.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2014-2024 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.ui.common.helper
+
+import android.net.Uri
+import io.getstream.chat.android.models.Attachment
+
+/**
+ * Generates a download URI for the given attachment.
+ */
+public fun interface DownloadAttachmentUriGenerator {
+
+    /**
+     * Generates a download URI for the given attachment.
+     *
+     * @param attachment The attachment to generate the download URI for.
+     *
+     * @return The download URI for the given attachment.
+     */
+    public fun generateDownloadUri(attachment: Attachment): Uri
+}
+
+// @InternalStreamChatApi
```

Comment:
Not sure if this was intentionally commented out, but the comment should probably be removed.

Reply:
Yep, it is not needed, removed